shell/platform/common/client_wrapper/event_channel_unittests.cc

```diff
@@ -274,4 +274,22 @@ TEST(EventChannelTest, HandlerOutlivesEventChannel) {
   EXPECT_EQ(on_cancel_called, true);
 }
 
+TEST(EventChannelTest, StreamHandlerErrorPassByValue) {
+  std::unique_ptr<StreamHandlerError<>> error = nullptr;
+
+  {
+    std::string code = "Code";
+    std::string msg = "Message";
+    std::unique_ptr<EncodableValue> details =
+        std::make_unique<EncodableValue>("Details");
+    error =
+        std::make_unique<StreamHandlerError<>>(code, msg, std::move(details));
```

Comment:
Let's add a second test that does a `(code, msg, nullptr)` so that we're absolutely certain that we aren't breaking that existing-in-the-wild pattern.